Add more colors

Mega-Cookie opened the pull request:
Hi. this is my first PR. hope I did it right.

Made these changes long before I even had a GH account.

They just add some more colors for objects and texts

Mega-Cookie committed:
Update index.html

Add some more colors for text and objects

Mega-Cookie committed:
Update index.html

ondras commented:
Hi @Mega-Cookie ,

thanks for the PR. It seems that you not only added more colors, but also deleted/changed some of the existing ones? If so, this does not seem to a be a backwards-compatible change, am I right?

Mega-Cookie committed:
Update index.html

Make backward compatible

Mega-Cookie commented:
I see what you mean. yeah I changed the hex codes of some existing colors. this would indeed not be backwards compatible. changed those back. should work now

ondras commented:
Great, thanks!
